src/content/language/methods.md

```diff
@@ -44,29 +44,30 @@ class Point {
 Operators are instance methods with special names.
 Dart allows you to define operators with the following names:
 
-|      |      |      |       |
-|------|------|------|-------|
-| `<`  | `+`  | `\|` | `>>>` |
-| `>`  | `/`  | `^`  | `[]`  |
-| `<=` | `~/` | `&`  | `[]=` |
-| `>=` | `*`  | `<<` | `~`   |
-| `-`  | `%`  | `>>` | `==`  |
+|       |      |      |      |       |      |
+|-------|------|------|------|-------|------|
+| `<`   | `>`  | `<=` | `>=` | `==`  | `~`  |
+| `-`   | `+`  | `/`  | `~/` | `*`   | `%`  |
+| `\|`  | `ˆ`  | `&`  | `<<` | `>>>` | `>>` |
+| `[]=` | `[]` |      |      |       |      |
```

Comment:
Why not the original dimensions? There's 20, 4x5 seems like the obvious choice

Reply:
Less space, honestly. Looked pretty strange for 4 columns across a broad page. I updated the order to the spec. It's not a big deal to me either way.

Reply:
Ok I'm fine with it, just checking